assets/components/bundle/bundle.jsx

```diff
@@ -3,7 +3,6 @@
 // ----- Imports ----- //
 
```

Comment:
There's not really any CSS that applies to this component that isn't page-specific.